### PR TITLE
Change Readme

### PR DESCRIPTION
**Pull request by adela-istrate-hs**
@valentin-radulescu-hs @andrei-picus-hs @flavius-tirnacop-hs 
Can you review this repo (not just the readme)?
Is this ok or do you have other ideas?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "chai": "3.5.0",
     "eslint": "3.0.1",
     "eslint-plugin-ubervu": "0.1.0",
+    "mugshot": "^0.2.0",
```
Comment (valentin-radulescu-hs) [inline package.json]: Mugshot isn't being used anywhere as far as I can tell so no need to include it here.

**Comment by valentin-radulescu-hs** [comment]
Please add a `.eslintrc` file so the linting actually works.

**Comment by flavius-tirnacop-hs** [comment]
@adela-istrate-hs Please make the file `.eslinrc.js` because `.eslintrc` was [deprecated](http://eslint.org/docs/user-guide/configuring#configuration-file-formats)

**Comment by flavius-tirnacop-hs** [comment]
LGTM

**Pull request merged**